# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../atl2/install.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- atl2-2.0.5/Makefile.vanilla	2008-08-14 10:38:23.000000000 +0200
+++ atl2-2.0.5/Makefile	2008-08-14 10:38:31.000000000 +0200
@@ -6,6 +6,9 @@
 all:
 	$(MAKE) -C $(KSRC) SUBDIRS=$(PWD) modules
 
+install:
+	$(MAKE) -C $(KSRC) SUBDIRS=$(PWD) modules_install
+
 clean:
 	rm -rf *.ko *.mod.* *.o .*.cmd .tmp_versions Module.symvers
 
